getting-started/gopher-run/leaderboard/leaderboard.go

```diff
@@ -55,19 +54,21 @@ func TopScores(ctx context.Context, client *firestore.Client) ([]ScoreData, erro
 }
 
 // AddScore adds a score to the leaderboard if it's in the top 10, or the scores database otherwise.
-func AddScore(ctx context.Context, client *firestore.Client, d ScoreData) error {
+func AddScore(ctx context.Context, client *firestore.Client, d ScoreData) (string, error) {
```

Comment:
What is the string return value? It's not obvious from the name of the function and not included in the function comment.

Reply:
Done